js/ai/src/formats/array.ts

```diff
@@ -24,21 +24,21 @@ export const arrayFormatter: Formatter<unknown[], unknown[]> = {
     contentType: 'application/json',
     constrained: true,
   },
-  handler: (request) => {
-    if (request.output?.schema && request.output?.schema.type !== 'array') {
+  handler: (schema) => {
+    if (schema && schema.type !== 'array') {
       throw new GenkitError({
         status: 'INVALID_ARGUMENT',
         message: `Must supply an 'array' schema type when using the 'items' parser format.`,
       });
     }
 
     let instructions: string | undefined;
-    if (request.output?.schema) {
+    if (schema) {
       instructions = `Output should be a JSON array conforming to the following schema:
     
-    \`\`\`
-    ${JSON.stringify(request.output!.schema!)}
-    \`\`\`
+\`\`\`
+${JSON.stringify(schema)}
+\`\`\`
```

Comment:
what's the plan for making these instructions configurable by the user? These formatters are not too complicated, are we saying "just write your own"?

Reply:
Users can do:

```ts
ai.generate({
  output: {
    format: 'array',
    instructions: false, // BYO instructions
    // or
    instructions: 'Output should be a JSON array.'
  }
});
```